models/transaction_logs.go

```diff
@@ -23,3 +28,14 @@ func TransactionLog(db *sql.DB, log TransactionLogs) error {
 		log.CreatedAt)
 	return err
```

Comment:
"JUST" return

Reply:
ok in fungsinya sekar ces jd gak gua ubah